lua/telescope/_extensions/marks.lua

```diff
@@ -6,47 +6,45 @@ local pickers = require("telescope.pickers")
 local conf = require("telescope.config").values
 local harpoon = require("harpoon")
 
-local function filter_empty_string(list)
```

Comment:
I removed that, because if you add invalid entry it would be filtered out from telescope view and you won't be able to deleted it from harpoon, even if it is there and is taking up index